src/iosrv/srv.c

```diff
@@ -224,9 +224,9 @@ dss_ult_wakeup(struct dss_sleep_ult *dsu)
 	}
 }
 
-/* Schedule the ULT(dtu->ult) and reschedule in @expire_secs seconds */
+/* Schedule the ULT(dtu->ult) and reschedule in @expire_secs micro seconds */
 void
-dss_ult_sleep(struct dss_sleep_ult *dsu, uint64_t expire_secs)
+dss_ult_sleep(struct dss_sleep_ult *dsu, uint64_t expire_nsecs)
```

Comment:
"_usecs" more align with micro seconds

Reply:
I will probably keeps using nano seconds here, since hlc and aggregation are using nano second to calculate its sleep time.  convert it to mtime might lose some accuracy, and also seems unnecessary.

Reply:
ok, just that in L229 it says "micro seconds".

Reply:
ah, ok, I will fix that. thanks

Reply:
Minor: Nanosecond doesn't make sense to me, probably microsecond or millisecond?